marketplace/core/types/channels/whatsapp_cloud/views.py

```diff
@@ -244,7 +244,7 @@ def get(self, request, project_uuid, *args, **kwargs):
             response.append(
                 {
                     "waba_id": app.config.get("wa_waba_id", None),
```

Comment:
Just a small suggestion: it's not necessary to pass None as the second argument to get() here, since dict.get("key") already returns None by default when the key is missing.